src/app/containers/Include/amp/Idt2Amp.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import AmpIncludeContainer from './Idt2Amp';
+
+const validIdt2Props = {
+  html: `<div>Hello</div>`,
+  type: 'idt2',
+  imageBlock: {
+    alt: 'image alt text',
+    height: 1864,
+    layout: 'responsive',
+    src: 'https://foobar.com/includes/image/816',
+    srcset:
+      'https://foobar.com/includes/image/470 470w,https://foobar.com/includes/image/816 816w',
+    width: 1632,
+  },
+};
+
+describe('AmpIncludeContainer', () => {
+  let container;
+  const includesBaseUrl = 'https://foobar.com/includes';
+  process.env.SIMORGH_INCLUDES_BASE_URL = includesBaseUrl;
+
+  beforeEach(() => {
+    jest.resetModules();
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  it('should render for a valid IDT2 include', async () => {
+    act(() => {
+      ReactDOM.render(<AmpIncludeContainer {...validIdt2Props} />, container);
```

Comment:
Is the extra container logic necessary? If we used react-testing-library `render()`, this will yield a container for us. It also handles `act()`.